### DISCUSSION TITLE
跟大家分享一下圖像辨識出錯的結果

### DISCUSSION THREAD

**Discussion by HuanYuuuuuu** [1101-GS4719-C/Python-Community-GS4719-C#573] [General]
[](url)
![S__4366341](https://user-images.githubusercontent.com/99785986/165514643-0ec3c318-e6ef-4b0a-b551-b99cfea33428.jpg)

![S__4366343](https://user-images.githubusercontent.com/99785986/165514658-daefc7b1-1622-4f17-beb2-9f8ad2227850.jpg)
![S__4366344](https://user-images.githubusercontent.com/99785986/165514666-778773e8-ecb0-4f86-900d-158fa2c640f2.jpg)

**Comment by PoooYen** [comment]
好好笑 看來還有很大的進步空間

**Comment by yi-1111** [comment]
看來要更大量的資料來建模增加辨識度

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期待未來的進步。

**Comment by d26488399** [comment]
笑死，是說狗狗好可愛

**Comment by Yuhsin000** [comment]
好好笑，狗狗要記得戴口罩喔XD

**Comment by Lai27330** [comment]
好有趣的結果，也希望未來對於低畫質圖片解析度的技術能更進步，讓使用者能透過識別系統得到更精確的結果。

**Comment by hapiwangy** [comment]
出錯的例子對於辨識也是重要的結果，可以方便我們改進模型

**Comment by yenwei12138** [comment]
原來倉鼠是長這樣，長知識了(誤
這應該是用來訓練模型的資料不足造成的吧

**Comment by cordydeng** [comment]
之前是有看過一個影片，牽著腳踏車去停車場亂玩車牌辨識系統，結果停車場的辨識系統分辨不出真假車牌，還蠻好笑的

**Comment by Rosancl** [comment]
好好笑，倉鼠超荒謬，但也說明建立大量模型訓練辨識的重要。

**Comment by lx3211** [comment]
看起來還要再多訓練一下ＸＤ
狗狗要記得戴口罩喔

**Comment by bloodtrail0511** [comment]
這裡有一隻超可愛的狗勾~

**Comment by henrywen547** [comment]
這真的也是頗好笑的。對於低畫質照片的樣本數增加或許有助於辨識的準確度

**Comment by ThomasMeng0515** [comment]
可以增加訓練模型的資料集，提升模型辨識的準確度，看來上面的模型都要好好改進才是 哈哈

**Comment by Jay-722** [comment]
下面那張圖真的在一堆地方都有看到，蠻白癡的

**Comment by Sazmafuji** [comment]
那條蛇可能其實是倉鼠
